README.rst

```diff
@@ -13,37 +13,40 @@ Nitrate - Test Case Management System
 .. image:: https://quay.io/repository/nitrate/nitrate/status
    :target: https://quay.io/repository/nitrate/nitrate/
 
-Nitrate is a new test plan, test run and test case management system, which is
-written in `Python`_ and `Django`_ web framework. It has a lot of great features,
-such as:
-
-* Easy to use to create and manage testing life cycle with plans, cases and runs.
-* Multiple and configurable authentication backend, e.g. Bugzilla and Kerberos.
+Nitrate is a new test plan, test run and test case management system,
+which is written in `Python`_ and `Django`_ (the Python web framework).
+It has a lot of great features, such as:
+
+* Ease of use in creating and managing test life cycles with plans,
+cases and runs.
```

Comment:
I'm not sure how to express accurately in English. For list items, the continuous lines should be formatted like this:

```
* Ease of use in creating and managing test life cycles with plans,
  cases and runs.
```

If you are using VIM, it can be done easily by pressing key sequence `gq<CR>`.